apps/research-service/src/validations/validation.controller.ts

```diff
@@ -69,4 +71,14 @@ export class ValidationController {
         pageNumber
       });
   }
+
+  @Post("validate")
+  @ApiOperation({
+    operationId: "validatePolygons",
+    summary: "Validate multiple polygons for various criteria"
+  })
+  @JsonApiResponse(ValidationResponseDto)
+  async validatePolygons(@Body() request: ValidationRequestDto): Promise<ValidationResponseDto> {
```

Comment:
This method is structured in a very RPC way. In v3 we really want to orient everything around resources instead. So instead of sending a POST to a method name ("validate"), this should be doing an action more like "create validations". 

Without digging into the service / controller too much, I'd like to see something more along the lines of:
```
@Post("polygonValidations")
@ApiOperation({ 
  operationId: "createPolygonValidations",
  summary: "Validation multiple polygons for various criteria"
})
@JsonApiResponse(ValidationDto, { hasMany: true })
```

A question here: Is it possible to have this endpoint return the same payload as the others in this controller? Can those be unified into a single resource?